tests/ga/test_update.py

```diff
@@ -1104,7 +1104,7 @@ def test_run_latest_passes_child_args(self):
         args = args[0]
 
         self.assertTrue(len(args) > 1)
-        self.assertTrue(args[0].startswith("python"))
+        self.assertRegex(args[0], r"(/.*/python[\d.]*)$", "The command doesn't contain full python path")
```

Comment:
A couple of minor quibbles: `[\d.]` might be better as `[\d\.]` in order to only match the `.` literal; you can preface the regex with a `^` to force the match to be the entire command (since it would then start with `^` and end with `$`).

Reply:
Agreed on using the `^`, will add it.

For the regex, looks like it still considers `.` as a literal rather than a wildcard.
```python

re.match(r"^(/.*/python[\d.]*)$", "/usr/bin/python3.6...").group()
'/usr/bin/python3.6...'

re.match(r"^(/.*/python[\d.]*)$", "/usr/bin/python3.6...qwqwq").group()
Traceback (most recent call last):
  File "<input>", line 1, in <module>
AttributeError: 'NoneType' object has no attribute 'group'
```